docs/src/install/install.rst

```diff
@@ -30,27 +31,26 @@ After downloading the Opaque codebase, build and test it as follows.
                    echo "export PATH=$PATH:/opt/spark/bin:/opt/spark/sbin" >> ~/.bashrc
                    source ~/.bashrc
 
-2. On the master, generate a keypair using OpenSSL for remote attestation.
-
-   .. code-block:: bash
-               
-                   openssl genrsa -out private_key.pem -3 3072
-
-3. Change into the Opaque root directory and edit Opaque's environment variables in ``opaqueenv`` if desired. Export Opaque and OpenEnclave environment variables via
+2. Change into the Opaque root directory and edit Opaque's environment variables in ``opaqueenv`` if desired. Export Opaque and OpenEnclave environment variables via
 
    .. code-block:: bash
                    
                    source opaqueenv
                    source /opt/openenclave/share/openenclave/openenclaverc
-
```

Comment:
Please add this line back (follows convention we established)

Reply:
Woops my bad - fixed!